ios/Runner/AppDelegate.swift

```diff
@@ -16,6 +17,11 @@ import Firebase
     }
     GMSServices.provideAPIKey("AIzaSyC5szIWBSfTg3SzJPkTPU7DPfZcdkvFd4A")
     GeneratedPluginRegistrant.register(with: self)
+    UIApplication.shared.setMinimumBackgroundFetchInterval(TimeInterval(60*15))
+    WorkmanagerPlugin.registerBGProcessingTask(withIdentifier: "trackingTask")
+    WorkmanagerPlugin.registerBGProcessingTask(withIdentifier: "scheduleDailyTasks")
+    // Register a periodic task in iOS 13+
+    WorkmanagerPlugin.registerPeriodicTask(withIdentifier: "be.tramckrijte.workmanagerExample.iOSBackgroundAppRefresh", frequency: NSNumber(value: 86400))
```

Comment:
i guess this is not required... it belongs to the example here https://github.com/fluttercommunity/flutter_workmanager/blob/d7e6ba5ef3796fafe54d9739e481f01058751b4d/example/ios/Runner/AppDelegate.swift#L23